src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -31,6 +32,9 @@
         private readonly Dictionary<string, HashSet<Result>> _ruleIdToResultsMap;
         private readonly Dictionary<string, RunMergingVisitor> _ruleIdToMergeVisitorsMap;
 
+        private static readonly string s_invalidFileNameCharRegexString = $"[{Regex.Escape(new string(Path.GetInvalidFileNameChars()))}]";
```

Comment:
>s_invalidFileNameCharRegexString [](http://example.com/codeflow?start=39&length=32)

This functionality seems very general and not something at all specific to merging SARIF logs. Should it be somewhere else? A file utilities class? Should it simply be an extension method somewhere?

Reply:
yes, moved the function to PathExtensions changed it to an extension method of string.